### DISCUSSION TITLE
are Multiple highlights on a card currently covered?

### DISCUSSION THREAD

**Discussion by andrewcrook** [st3v3nmw/obsidian-spaced-repetition#373] [Q&A] [answered]
are Multiple highlights on a card currently covered? 

e.g this example has two

`this is a ==test1== and a test number ==two== but no test three`

**Comment by st3v3nmw** [comment, marked as answer]
Yes, multiple clozes work. If the 2nd one doesn't show up, you might have the "bury related/sibling cards" setting turned on.

**Comment by kennystocker** [comment]
I am new to this so may have something wrong in my set up, but when I try Andrews example it generates 2 cards. The first card has a blank for the first highlight but renders the second highlight. The second card then shows a blank for the second highlight. Is that correct behaviour? (I have bury related siblings off).
![Screenshot 2022-03-10 at 00 36 56](https://user-images.githubusercontent.com/25698258/157563657-76336582-c022-4483-83d8-6e88d50e3ec7.png)
.

**Reply by st3v3nmw** [reply]
Yes, that's the expected behavior.

**Reply by noamsiegel** [reply]
Is there a way to make it so that all clozes are hidden until the answer is shown? @st3v3nmw

**Reply by st3v3nmw** [reply]
@noamsiegel, currently no it's not possible. I have a plan to extend cloze syntax to match Anki's
